.github/workflows/python_tests.yml

```diff
@@ -116,16 +116,19 @@ jobs:
       - name: Get build dependencies
         working-directory: ./sdks/python
         run: pip install -r build-requirements.txt
+      - name: Generate protos
+        working-directory: ./sdks/python
+        run: python gen_protos.py
       - name: Install tox
-        run: pip install tox==3.27.1
+        run: pip install tox
```

Comment:
Should we still pin to a 4.x version to avoid this possibly breaking when tox 5.x is released?